backend/api/Database/Models/Mission.cs

```diff
@@ -103,6 +104,27 @@ public static MissionStatus MissionStatusFromString(string status)
                   )
             };
         }
+
+        public void CalculateEstimatedDuration()
+        {
+            decimal robotVelocity = (decimal)1.5 * 1000 / 60; // km/t => m/min
+            decimal efficiencyFactor = (decimal)0.20;
+            decimal inspectionTime = 2; // min/tag
```

Comment:
I think doubles are plenty good enough here, no need to introduce another data type?

Reply:
I got overflow when dividing using "/" but changing to the function decimal.Devide solved the problem